File: exchange-container/client/src/main/java/db/MongoDB.java
package db;

import com.mongodb.rx.client.MongoClients;
import com.mongodb.rx.client.MongoCollection;
import com.mongodb.rx.client.MongoDatabase;
import org.bson.Document;

public class MongoDB {
    private static final MongoDatabase database
        = MongoClients.create("mongodb://localhost:27017")
        .getDatabase("client");

    public static MongoCollection<Document> getUsersEmptyCollection() {
        MongoCollection<Document> users = database.getCollection("users");
        users.drop().toBlocking().single();
        return users;
    }
}

File: event-sourcing-fitness/src/main/kotlin/db/MongoDB.kt
package db

import com.mongodb.rx.client.MongoClients
import com.mongodb.rx.client.MongoCollection
import org.bson.Document

object MongoDB {
    private val database = MongoClients
        .create("mongodb://localhost:27017")
        .getDatabase("fitness-center")

    fun dropDatabaseBlocking() {
        database.drop().toBlocking().single()
    }

    fun getSubscriptions(): MongoCollection<Document> {
        return database.getCollection("subscriptions")
    }

    fun getVisits(): MongoCollection<Document> {
        return database.getCollection("visits")
    }

}